Ticket - 91638 zMessage to alert user if prep item was disposed

sewa67 opened the pull request:
Please review this change and merge into development branch 
FIles - est/BuildDefaultPrep.p
jc/jc-calc.p
system/PrepProcs.p
oe/ordlmisc.p

sewa67 committed:
Ticket - 91638 zMessage to alert user if prep item was disposed
FIles - est/BuildDefaultPrep.p
jc/jc-calc.p
oe/OrderProcs.p
oe/ordlmisc.p

jay-advantzware reviewed (commented):
Will this only display when the item is initially added - Would not want this to appear every time an order is updated, only on initial add.

sewa67 committed:
Ticket - 91638 zMessage to alert user if prep item was disposed
Files - FIles - est/BuildDefaultPrep.p
jc/jc-calc.p
system/PrepProcs.p
oe/ordlmisc.p
